routes/auth.route.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose');
+
+require('../models/users.models');
+var User = mongoose.model('Users');
+
+
+
+module.exports = function(app, passport){
+   function linkedinSignInCallback(req, res, next) {
+    passport = req._passport.instance;
+    passport.authenticate('linkedin',function(err, user, token) {
+      if(err) {
+        return next(err);
+      }
+      if(!user) {
+        return res.redirect('http://localhost:8000');
+      }
+        return res.redirect('http://localhost:800f0');
```

Comment:
Typo I guess? Correct. And check the logic here. I'm not sure I understand what you are trying to do.